web/src/message_events.ts

```diff
@@ -809,6 +810,12 @@ export function update_messages(events: UpdateMessageEvent[]): void {
                 post_edit_topic,
                 post_edit_stream_id,
             );
+        } else {
```

Comment:
This check is wrong -- we should be positively checking for a content edit, not doing an else clause like this.